nemo_aligner/algorithms/ppo.py

```diff
@@ -348,15 +349,15 @@ def run_training(self, dataloader_iter):
         return loss_mean, metrics
 
     def fit(self):
-        if (not isinstance(self.train_dataloader.batch_sampler, MegatronPretrainingRandomBatchSampler)) and (
+        if (not isinstance(self.train_dataloader.batch_sampler, MegatronPretrainingRandomSampler)) and (
```

Comment:
Do we still need this check considering that the code will now always instantiate a `MegatronPretrainingRandomSampler`?

Reply:
It's probably a good idea to leave it for now for maximum safety and we can remove it later

Reply:
Ok, not a big deal, we can keep it for now => resolving.